sys/net/gnrc/network_layer/ndp/node/gnrc_ndp_node.c

```diff
@@ -75,7 +78,6 @@ kernel_pid_t gnrc_ndp_node_next_hop_l2addr(uint8_t *l2addr, uint8_t *l2addr_len,
     if (!ipv6_addr_is_link_local(dst)) {
```

Comment:
I know it's good practice to put all variable definitions at the beginning of the function, but I am more inclined to have `ipv6_addr_t next_hop_actual;` one line above this line. This way you won't have to add the additional and ugly `#ifdef MODULE_FIB`.